docs/generated/packages/node/generators/library.json

```diff
@@ -51,31 +51,37 @@
       "skipFormat": {
         "description": "Skip formatting files.",
         "type": "boolean",
-        "default": false
+        "default": false,
+        "x-priority": "internal"
       },
       "skipTsConfig": {
         "type": "boolean",
         "default": false,
-        "description": "Do not update `tsconfig.base.json` for development experience."
+        "description": "Do not update `tsconfig.base.json` for development experience.",
+        "x-priority": "internal"
       },
       "publishable": {
         "type": "boolean",
-        "description": "Create a publishable library."
+        "description": "Create a publishable library.",
+        "x-priority": "important"
       },
       "buildable": {
         "type": "boolean",
         "default": false,
-        "description": "Generate a buildable library."
+        "description": "Generate a buildable library.",
+        "x-priority": "important"
       },
       "compiler": {
         "type": "string",
         "enum": ["tsc", "swc"],
         "default": "tsc",
-        "description": "The compiler used by the build and test targets."
+        "description": "The compiler used by the build and test targets.",
+        "x-priority": "important"
       },
       "importPath": {
         "type": "string",
-        "description": "The library name used to import it, like `@myorg/my-awesome-lib`. Must be a valid npm name."
+        "description": "The library name used to import it, like `@myorg/my-awesome-lib`. Must be a valid npm name.",
```

Comment:
Doesn't this have a strong, sensible default? Personally, I don't change the import path most of the time. Whats the reason for it being important here?

Reply:
This is just me; I'm in the habit of setting `importPath` due to me creating publishable libraries all the time. In the context of `x-priority`, `important` can be taken off